### ISSUE TITLE
Show the ALT text in the larger size view

### ISSUE THREAD

**Issue by Ember-ruby** [mastodon/mastodon#33804]
### Pitch

Mastodon has recently added a feature where you can click on the ALT label to view the alt text, no such feature exists when you click on an image however

as clicking on an image means you want to see more information about it, i think the alt text should be displayed there by default, like on bluesky

### Motivation

the regular benefits of alt text, showing it by default would help ease of access and normalisation of using alt text
